crates/model/src/solver_competition.rs

```diff
@@ -0,0 +1,40 @@
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+// These types are going to be updated once the openapi documentation gets
+// merged. Consider them a placeholder.
+
+pub type SolverCompetitionResponse = HashMap<String, SolverSettlement>;
```

Comment:
Shouldn't this be a `Vec<(String, SolverSettlement)>` if we want to allow multiple settlements from the same solver (which, AFAIU, we currently do right - because of settlement merging?).

Reply:
Like I wrote above there, this is a placeholder until we merge the Openapi PR. Then I will make sure the types are right.